src/System.Management.Automation/engine/NativeCommandProcessor.cs

```diff
@@ -1797,8 +1797,7 @@ internal void Start(Process process, NativeCommandIOFormat inputFormat)
             //Get the encoding for writing to native command. Note we get the Encoding
             //from the current scope so a script or function can use a different encoding
             //than global value.
-            Encoding pipeEncoding = _command.Context.GetVariableValue(SpecialVariables.OutputEncodingVarPath) as System.Text.Encoding ??
-                                    Encoding.ASCII;
+            Encoding pipeEncoding = _command.Context.GetVariableValue(SpecialVariables.OutputEncodingVarPath) as System.Text.Encoding ??  Utils.iso8859;
```

Comment:
Minor comment: there is an extra space before `Utils.iso8859`.

Reply:
fixed!